hack/verify-import-order.sh

```diff
@@ -20,11 +20,11 @@ cd $(dirname $0)/..
 source hack/lib.sh
```

Comment:
I belive you didn't want to push this file, right?

Reply:
Yes, I did want to push it. 
I need to rebase, I did a specific PR yesterday for that. https://github.com/kubermatic/kubermatic/pull/10720.
Once rebased, it will go away.